src/store/reducers/authApiStatusReducer.js

```diff
@@ -0,0 +1,12 @@
+import * as types from '../actions/actionTypes';
+import initialState from './initialState';
+
+
+const authApiStatusReducer = (state = initialState.authApiStatus, action) => {
+    if (action.type === types.AUTH_API_STATUS__SUCCESS) {
+        return action.payload?.status;
```

Comment:
It's unlikely that `action.payload` will be undefined here, but if we do want to account for that with the optional chaining, as implemented, then I think it should also coalesce to `null`, since Redux complains about assigning `undefined` to slices of the store.
```suggestion
        return action.payload?.status ?? null;
```